solver/src/settlement.rs

```diff
@@ -328,6 +335,30 @@ fn sell_order_surplus(
     }
 }
 
+#[cfg(test)]
```

Comment:
ubernit: Maybe put this in `pub mod tests`? I'm also totally fine with leaving it here, so feel free to ignore.